### PR TITLE
rewrite Readme (pip, rst, etc)

### PR DESCRIPTION
**Pull request by thorade**

### CODE REVIEW

**Comment by bilderbuchi** [comment]
could probably be simplified, pip supports git natively: https://pip.pypa.io/en/stable/reference/pip_install/#git

**Comment by thorade** [comment]
You are right, just for installing one doesn't have to clone it.

**Comment by bilderbuchi** [comment]
Would also work with `pip install -U git+https://github.com/OpenModelica/jupyter-openmodelica.git` - not involving zip on the github side would maybe be a tad faster.

**Comment by thorade** [comment]
But using the zip will also work for people that do not have git installed.

**Comment by thorade** [comment]
I added more changes and switched from markdown to rst,
just in case this gets published to PyPi.

**Comment by thorade** [comment]
Done.
Can you use Squash&Merge please.

And probably close
https://trac.openmodelica.org/OpenModelica/ticket/3705

**Comment by bilderbuchi** [comment]
> But using the zip will also work for people that do not have git installed.

Good point!

**Comment by bilderbuchi** [comment]
While you're proposing changes, I found that the dependency on `numpy` is not reflected in setup.py's `install_requires`.

**Comment by thorade** [comment]
That should probably go into a separate issue and/or PR. Will you do that?
Also, OMPython already requires numpy, so it should never become a problem.

**Comment by thorade** [comment]
Could this be merged?

**Comment by thorade** [comment]
@dietmarw could you review whether these instructions are more clear / clear enough?
https://trac.openmodelica.org/OpenModelica/ticket/3705

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+- `Python <https://www.python.org/downloads/>`_
+- `Jupyter Notebook <https://jupyter.readthedocs.org/en/latest/install.html>`_
+- `OpenModelica <https://openmodelica.org>`_ v1.12.0 or newer
+- `OMPython <https://github.com/OpenModelica/OMPython>`_.
+
+When using an existing Python installation, it is strongly recommended to update ``pip`` and ``jupyter`` by running::
+
+  python -m pip install -U pip setuptools wheel
+  python -m pip install -U jupyter
```
Comment (dietmarw) [inline README.rst:21]: Why the update of setuptools and wheel ?

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  jupyter notebook
+  # In the notebook interface, select OpenModelica from the 'New' menu
+
+The repository also contains the
+`example notebook file OpenModelica.ipynb <https://github.com/OpenModelica/jupyter-openmodelica/blob/master/OpenModelica.ipynb>`_
+
+The plotting is done with the help of Dygraph javascript.
+If plotting does not works,
+make sure the "dygraph-combined.js" is in the current working directory.
```
Comment (dietmarw) [inline README.rst]: typo "works" --> "work"

**Comment by dietmarw** [comment]
I've added som comments. But honestly I don't see the point in the way the openmodelica jupyter notebook is implemented. Especially the plot dependencies on some javascript library. I'd rather prefer to simply use OpenModelica from within a normal Python Jupyter Notebook where one gets to use the power of the whole Python world (especially Matplotlib) and this is possible thanks to OMPython.

File: README.rst (hunk 1 above)
Comment (thorade) [reply README.rst:21]: To make sure numpy is installed from a wheel and not compiled from source.

File: README.rst (hunk 2 above)
Comment (thorade) [reply README.rst]: fixed

File: README.rst (hunk 1 above)
Comment (thorade) [reply README.rst:21]: also see https://github.com/travis-ci/travis-ci/issues/2650#issuecomment-252285413

**Comment by thorade** [comment]
replaced by #14

**Pull request closed without merging**